sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsUtilClient.java

```diff
@@ -805,11 +804,48 @@ private AbstractiveSummarizationTaskParameters getAbstractiveSummarizationTaskPa
             try {
                 return operationResult.onErrorMap(Utility::mapToHttpResponseExceptionIfExists);
             } catch (RuntimeException ex) {
-                return monoError(logger, ex);
+                return monoError(LOGGER, ex);
             }
         };
     }
 
+    private Function<PollingContext<AnalyzeActionsOperationDetail>, AnalyzeActionsOperationDetail>
+        activationOperationLanguageApiSync(Iterable<TextDocumentInput> documents, TextAnalyticsActions actions,
+            Context context) {
+        return pollingContext -> {
+            final ResponseBase<AnalyzeTextsSubmitJobHeaders, Void> analyzeResponse =
+                service.submitJobWithResponse(
+                    new AnalyzeTextJobsInput()
+                        .setDisplayName(actions.getDisplayName())
+                        .setAnalysisInput(new MultiLanguageAnalysisInput()
+                            .setDocuments(toMultiLanguageInput(documents)))
+                        .setTasks(getAnalyzeTextLROTasks(actions)),
+                    context);
```

Comment:
Have we covered the tracing context for this client .addData(AZ_TRACING_NAMESPACE_KEY, COGNITIVE_TRACING_NAMESPACE_VALUE)
?

Reply:
Yes. It is already covered in https://github.com/Azure/azure-sdk-for-java/pull/32867/files/af932a4d48a720adb3d47a602cb5d2169202714a#diff-89665ffff561558826bea5cd11890970fb58f56bb17d20c2de3b1cb05b282c26R335